### DISCUSSION TITLE
Mobs that spawn during boss fights should be inactive for a short time after spawning.

### DISCUSSION THREAD

**Discussion by UndoUreche** [chromiecraft/chromiecraft#6860] [Suggestions]
We have seen how painful this is in SSC, but this is a more general issue. 

Since many boss fights will spawn adds and plenty of those will do so in random positions and/or at random times, it would be best to have all mobs that spawn during boss fights be made inactive or disarmed for something like 0.5s. This would prevent them from immediately killing the person they spawn on top of.

An example of this, if Vashj's striders acting like thunderbolts, randomly striking a clothie straight into a repair bill because they spawned on top of them and hit them without warning in frame 1.

**Comment by balleny** [comment]
Vashjs striders and Nagas, as well as the adds in the Lurker fight actually have a fixed path they run before they engage (if its getting fixed at some point)

most other mobs that "just spawn" have a 2 second idle time on retail before they start attacking (some may engage already after 1 second if they are being attacked already). this would e.g. apply for Hydross adds or the adds spawned during the Karathress fight

Azerothcore as a whole lacks this "idle on summon" for literally anything because its a generic issue and was never fully addressed yet. Often resulting in one-shots on ChromieCraft given the custom tuning

**Reply by UndoUreche** [reply]
That's precisely why I called it a more general issue. Also, since there's a class for creatures, I'm assuming one can extend/decorate it to simply add that behavior to any such mobs without too much fuss. 

Although for something as big as acore there will always be risks with such sweeping changes. I'm scared we may see adds break in raids we've progressed past and there will be nobody to find out about it.

**Reply by UndoUreche** [reply]
I suppose the point of this discussion is for people to chime in, and help explore the ratio of effort (only the devs can quantify this) to benefit (player happiness), and maybe make a good case for why it should be implemented.

**Reply by balleny** [reply]
you can also see it here quite well that Nagas engage very quickly after their path, directly killing a hunter very close to their path on spawn
https://youtu.be/FEYAX7l5H6A?t=2544

a better view on their pathing can be seen here https://youtu.be/FEYAX7l5H6A?t=2588

**Reply by Wheemix** [reply]
Shouldnt this rather be a bug report than a suggestion? As far as i know the same is true for every regular respawn in the open world too so i definitely agree it's an important issue

**Reply by balleny** [reply]
yes, in the end it should be a bugreport and tbh. it should change the default for anything that is being summoned as this would be the lesser harm.

alternatively, you can only go and make individual adjustments to each and every spawn to be delayed which is nonsense

**Reply by balleny** [reply]
any type of "summon npc", especially inside dungeons and raids, should include the idle time

**Reply by shaszun** [reply]
> Vashjs striders and Nagas, as well as the adds in the Lurker fight actually have a fixed path they run before they engage (if its getting fixed at some point)

That's not exactly true. They should be moving towards Vashj, but if you have someono in their hit range or a healer in nearby vicinity healing in the moment of the spawn, they will be aggroed. All you currently have to do, is either stay away from the 3-4 spawn points and hope you have a tank in the vicinity with good reaction to taunt it.

**Reply by shaszun** [reply]
I had reported the Hydross adds way before SSC release and it's still not even confirmed as an issue. Yea, this is generic issue, but if they don't want to compromise the old content in classic in some way. They will need to find a way to apply it for TBC and in the future wotlk, and when someone report it for classic content, just to be easaly included.
